deltacat/tests/compute/test_janitor.py

```diff
@@ -130,21 +137,27 @@ def test_janitor_handles_multiple_timed_out_transactions(self, temp_dir):
         os.makedirs(failed_txn_dir, exist_ok=True)
 
         # Create multiple timed-out transaction logs
-        txn_ids = ["txn_one", "txn_two", "txn_three"]
         start_time = time.time_ns() - 1_000_000_000  # 1 second in the past
```

Comment:
Do we need to change this value `1_000_000_000` to `500_000_000`as well? Maybe use a constant value to make sure it's consistent instead.

Reply:
Agreed - this will make the test more resilient to potential system clock skew on different hosts (although we can't go too far back in time, since we now have sanity checks making sure that the start/end transaction times look like a reasonable recent date since someone may have conceivably started using DeltaCAT, so I've fixed them to a starting point of January 1, 2025).